src/Razor/src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/BackgroundDocumentProcessedPublisher.cs

```diff
@@ -81,10 +81,10 @@ public override void DocumentProcessed(OmniSharpDocumentSnapshot document)
 
             _foregroundDispatcher.AssertForegroundThread();
 
-            if (!FileKinds.IsComponent(document.FileKind) || FileKinds.IsComponentImport(document.FileKind))
+            if (FileKinds.IsComponentImport(document.FileKind))
```

Comment:
What about _ViewImports.cshtml? We don't want to publish them right?

Reply:
Doesn't matter if we do, it's just extra work. I mention that in the comment below. I could technically inspect the document path and do a hard check against it I guess

Reply:
Ah never mind I didn't read the comment fully. My bad.